app/src/main/java/com/kmarlow/custominstrumentation/InstApplication.java

```diff
@@ -1,15 +1,34 @@
 package com.kmarlow.custominstrumentation;
 
-import android.app.Application;
+import android.content.Context;
 import android.content.Intent;
+import android.support.multidex.MultiDexApplication;
 
+import com.kmarlow.custominstrumentation.sdk.AndromiumControllerService;
+import com.squareup.leakcanary.LeakCanary;
+import com.squareup.leakcanary.RefWatcher;
 
-public class InstApplication extends Application {
+
+public class InstApplication extends MultiDexApplication {
```

Comment:
This doesn't need to be multidex application now that we're providing the framework instead of compiling it.